app/models/cache-model.js

```diff
@@ -85,17 +73,21 @@ function getSurveyHashes(survey) {
         } else {
             const key = _getKey(survey);
 
-            client.hmget(key, ['formHash', 'xslHash'], (error, hashArr) => {
-                if (error) {
-                    reject(error);
-                } else if (!hashArr || !hashArr[0] || !hashArr[1]) {
-                    resolve(survey);
-                } else {
-                    survey.formHash = hashArr[0];
-                    survey.xslHash = hashArr[1];
-                    resolve(survey);
+            cacheClient.hmget(
+                key,
+                ['formHash', 'xslHash'],
+                (error, hashArr) => {
+                    if (error) {
+                        reject(error);
+                    } else if (!hashArr || !hashArr[0] || !hashArr[1]) {
+                        resolve(survey);
+                    } else {
+                        survey.formHash = hashArr[0];
+                        survey.xslHash = hashArr[1];
+                        resolve(survey);
+                    }
```

Comment:
Did anything change here?

Reply:
`client` -> `cacheClient`, then auto-formatting. I noticed today that GitHub (recently?) added a diff option to ignore whitespace changes. It isn't perfect, but it did help me see the change more clearly in this case.